packages/core/core/src/applyRuntimes.js

```diff
@@ -128,7 +138,7 @@ export default async function applyRuntimes({
       }
     }, runtimeNode);
 
-    runtimesGraph.traverse((node, _, actions) => {
+    runtimesGraph._graph.traverse((node, _, actions) => {
```

Comment:
`runtimesGraph` is now a BundleGraph (which has a Graph), not just a Graph.

I can't remember if BundleGraph ever had a traverse method itself, but it would make sense to have one.